src/app/layouts/layouts-routes-data.ts

```diff
@@ -32,6 +32,11 @@ export const layoutsRoutesData = {
     'splitter-horizontal-sample': { displayName: 'Splitter Horizontal', parentName: 'Splitter' },
     'splitter-nested-sample': { displayName: 'Splitter Nested Panes', parentName: 'Splitter' },
     'splitter-styling-sample': { displayName: 'Splitter Styling', parentName: 'Splitter' },
+    'stepper-overview-sample': { displayName: 'Stepper 1', parentName: 'Stepper' },
```

Comment:
It's best to make the display name be the same as the name of the sample component because it's more descriptive.